packages/vapor/scss/sprites.scss

```diff
@@ -1,78 +1 @@
-}
-i.coveo-sprites-misc-filter,
-.coveo-sprites-misc-filter {
-    width: 12px;
-    height: 12px;
-    text-indent: 12px;
-    background-position: -46px 0px;
-}
-i.coveo-sprites-misc-alert,
-.coveo-sprites-misc-alert {
-    width: 11px;
-    height: 11px;
-    text-indent: 11px;
-    background-position: -60px 0px;
-}
-i.coveo-sprites-misc-panel-expended,
-.coveo-sprites-misc-panel-expended {
-    width: 10px;
-    height: 10px;
-    text-indent: 10px;
-    background-position: -73px 0px;
-}
-i.coveo-sprites-misc-panel-collapsed,
-.coveo-sprites-misc-panel-collapsed {
-    width: 10px;
-    height: 10px;
-    text-indent: 10px;
-    background-position: -85px 0px;
-}
-i.coveo-sprites-tables-sort_desc,
-.coveo-sprites-tables-sort_desc {
-    width: 6px;
-    height: 6px;
-    text-indent: 6px;
-    background-position: -97px 0px;
-}
-i.coveo-sprites-tables-sort_asc,
-.coveo-sprites-tables-sort_asc {
-    width: 6px;
-    height: 6px;
-    text-indent: 6px;
-    background-position: -97px -8px;
-}
+.coveo-sprites-misc-closepopup, .coveo-sprites-misc-spinner_arrow_up, .coveo-sprites-misc-spinner_arrow_down, .coveo-sprites-misc-filter, .coveo-sprites-misc-alert, .coveo-sprites-misc-panel-expended, .coveo-sprites-misc-panel-collapsed, .coveo-sprites-tables-sort_desc, .coveo-sprites-tables-sort_asc{background-image:url("../dist/images/CoveoStyleGuide.Sprites.png");display:inline-block;background-repeat:no-repeat;overflow:hidden;background-size: 103px 16px;}i.coveo-sprites-misc-closepopup, .coveo-sprites-misc-closepopup {      background-position:0px 0px;      width:16px;      height:16px;      text-indent:16px;    }i.coveo-sprites-misc-spinner_arrow_up, .coveo-sprites-misc-spinner_arrow_up {      background-position:-18px 0px;      width:12px;      height:12px;      text-indent:12px;    }i.coveo-sprites-misc-spinner_arrow_down, .coveo-sprites-misc-spinner_arrow_down {      background-position:-32px 0px;      width:12px;      height:12px;      text-indent:12px;    }i.coveo-sprites-misc-filter, .coveo-sprites-misc-filter {      background-position:-46px 0px;      width:12px;      height:12px;      text-indent:12px;    }i.coveo-sprites-misc-alert, .coveo-sprites-misc-alert {      background-position:-60px 0px;      width:11px;      height:11px;      text-indent:11px;    }i.coveo-sprites-misc-panel-expended, .coveo-sprites-misc-panel-expended {      background-position:-73px 0px;      width:10px;      height:10px;      text-indent:10px;    }i.coveo-sprites-misc-panel-collapsed, .coveo-sprites-misc-panel-collapsed {      background-position:-85px 0px;      width:10px;      height:10px;      text-indent:10px;    }i.coveo-sprites-tables-sort_desc, .coveo-sprites-tables-sort_desc {      background-position:-97px 0px;      width:6px;      height:6px;      text-indent:6px;    }i.coveo-sprites-tables-sort_asc, .coveo-sprites-tables-sort_asc {      background-position:-97px -8px;      width:6px;      height:6px;      text-indent:6px;    }
```

Comment:
Revert this..?

Reply:
No, this file is generated at compile time and prettier was then re-formatting it, causing `packages/vapor/scss/sprites.scss` to always appear in the diff. I added its path to `.prettierignore` so it should be the last time we see this diff in theory :slightly_smiling_face: